docusaurus/docs/dev-docs/backend-customization/models.md

```diff
@@ -649,7 +649,7 @@ Lifecycle hooks are functions that take an `event` parameter, an object with the
 | Key      | Type              | Description                                                                                                                                                      |
 | -------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `action` | String            | Lifecycle event that has been triggered (see [list](#available-lifecycle-events))                                                                                |
-| `model`  | Object            | Model object                                                                                                                                                       |
+| `model`  | Array of strings (uid)            | An array of uids of the content-types or single-types you want to listen for events for. Not supplying this argument means it will listen on all models |
```

Comment:
```suggestion
| `model`  | Array of strings (uid)            | An array of uids of the content-types whose events will be listened to.<br />If this argument is not supplied, events are listened on all content-types. |
```

Single types are content-types. I assume you meant collection types or single types, so we can simplify using the generic "content types" wording.